screens/InventoryByCategoryScreen.js

```diff
@@ -39,17 +44,37 @@ export default class InventoryByCategoryScreen extends React.Component {
           inventory: inventory
         });
       }.bind(this));
+
+    await firebase
```

Comment:
Let's not do this -- instead, let's pass the categoryDisplayName like we pass category, via "this.props.navigation.state.params.category".

This value is set from the HomeScreen onClick method. We save a firebase call.